statsmodels/tsa/vector_ar/tests/test_var.py

```diff
@@ -11,6 +11,8 @@
 import sys
 
 import numpy as np
+import pandas as pd
+from pandas.testing import assert_index_equal
```

Comment:
pretty much everywhere else we import from pandas.util.testing (and im not sure if we ever use statsmodels.compat.pandas.ptesting).  ill take a look in follow-up.